src/ILVerification/tests/ILTests/InterfaceImplementationInherit.il

```diff
@@ -0,0 +1,66 @@
+.assembly InterfaceImplementationInherit
```

Comment:
I do not think this needs a new file. This can be in one the existing ones.

We prefer to have larger test assemblies with multiple tests.